client/my-sites/customer-home/main.jsx

```diff
@@ -56,6 +58,13 @@ const Home = ( {
 		);
 	}
 
+	if ( 'purchase-success' === noticeType ) {
+		const successMessage = translate( 'Your purchase has been completed!' );
+		notices.success( successMessage, {
```

Comment:
There's currently a project to Reduxify the `notices` module (see https://github.com/Automattic/wp-calypso/issues/48408) (cc @tyxla) so we should probably do this with `const reduxDispatch = useDispatch();` and `reduxDispatch( successNotice( /* ... */ ) )` instead. Be aware that the `persistent` option has become `isPersistent`, which tripped me up before.

Reply:
Thanks for addressing this early @sirbrillig @niranjan-uma-shankar, the old notices module is almost gone now (and deprecated as per [its README](https://github.com/Automattic/wp-calypso/tree/trunk/client/notices#notices)) so Ideally we shouldn't introduce new instances of it.  👍